enterprise/app/services/enterprise/billing/handle_stripe_event_service.rb

```diff
@@ -73,15 +78,15 @@ def current_plan_credits
   def update_account_attributes(subscription, plan)
     # https://stripe.com/docs/api/subscriptions/object
     account.update(
-      custom_attributes: {
-        stripe_customer_id: subscription.customer,
-        stripe_price_id: subscription['plan']['id'],
-        stripe_product_id: subscription['plan']['product'],
-        plan_name: plan['name'],
-        subscribed_quantity: subscription['quantity'],
-        subscription_status: subscription['status'],
-        subscription_ends_on: Time.zone.at(subscription['current_period_end'])
-      }
+      custom_attributes: account.custom_attributes.merge(
```

Comment:
This was overriding, credit usage and hence reseting the usage, so updated it to update existing values